octodns_dyn/__init__.py

```diff
@@ -2,21 +2,27 @@
 #
 #
 
-from collections import defaultdict
-from dyn.tm.errors import DynectGetError
+import collections.abc  # noqa: E402
+# dyn libs needs the folowing alias as it's trying to use a
+# deprecated feature that worked up to python 3.9, but doesn't
+# in 3.10
+collections.Iterable = collections.abc.Iterable
```

Comment:
Would it be possible to include everything you can normally w/o the `noqa` bits and then do the stuff that causes the need for it as the last bit so that only those few lines require it.

My guess/hope is that this collections line and some/all of the `from dyn*` bits a few lines down could all be moved after the other imports so that only the `from dyn*` lines need the marker.

Reply:
Yep, I've reordered them. The git diff is a little wonky looking, but it all seems to lint/test now with fewer # noqa: E402 lines.